.github/workflows/release_candidate_validation.yml

```diff
@@ -46,18 +46,6 @@ jobs:
             flinkProfile: 'flink1.16'
             sparkProfile: 'spark3.3'
             sparkRuntime: 'spark3.3.1'
-          - scalaProfile: 'scala-2.12'
```

Comment:
Could you apply the same set of changes from #11653 on the RC validation workflow?